tasks/push-disk-images/push-disk-images.yaml

```diff
@@ -37,6 +40,9 @@ spec:
         # .cdn.env is likely to change in the future. This is just for POC
         env=$(jq -r '.cdn.env' "$(workspaces.data.path)/$(params.dataPath)")
 
+        RESULTS_FILE="$(workspaces.data.path)/$(params.resultsDirPath)/push-disk-images-results.json"
+        jq -c '{files: [.components[].staged.files[].filename]}' <<< "$snapshot" > "$RESULTS_FILE"
```

Comment:
Can you name this something more specific than files? You can see here https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/create-advisory/create-advisory.yaml#L189 how we do it for advisories. Maybe something like `disk-image-files` or anything to relate it to the task?